configs/pulpcore/3.28.yaml

```diff
@@ -0,0 +1,138 @@
+---
+:project: pulpcore
+:github_org: pulp
+:nightly: false
+:repos: []
+:releases:
+  :3.28:
+    :redmine_version_id: -1
+:mash_scripts:
+  - pulpcore-3.28-el8
+  - pulpcore-3.28-el9
+:strict_keys: true
+:gpg_key: 2622DB63
+:tags:
+  - name: pulpcore-3.28-el8
+    based_off: pulpcore-3.22-el8
+    helper_tags:
+      pulpcore-3.28-el8-override: null
+    build_target: pulpcore-3.28-el8-build
+    build_package_group_source_tag: pulpcore-3.28-el8-build
+    arches:
+      - x86_64
+    inherits:
+      pulpcore-3.28-el8-build:
+        pulpcore-3.28-el8-override: 0
+      pulpcore-3.28-el8-override:
+        pulpcore-3.28-el8: 0
+    external_repos:
+      - name: flat-base-el8
+        mode: bare
+      - name: flat-python36-el8
+        mode: bare
+      - name: flat-python39-el8
+        mode: bare
+      - name: flat-python39-devel-el8
+        mode: bare
+      - name: flat-ruby27-el8
+        mode: bare
+      - name: flat-swig30-el8
+        mode: bare
+      - name: flat-rust-toolset-el8
+        mode: bare
+      - name: flat-llvm-toolset-el8
+        mode: bare
+      - name: centos8-devel
+        mode: bare
+      - name: centos8-configmanagement-ansible-29
+        mode: bare
```

Comment:
I thought we'd stop using this now that ansible-core is in EL8.

Reply:
That's correct, we should drop this and use ansible-core.